perma_web/conftest.py

```diff
@@ -662,3 +676,58 @@ def one_two_three_dict():
     assert 'three' in data
     assert 'four' not in data
     return data
+
+
+#
+# Helpers
+#
+
+def randomize_capitalization(s):
+    return ''.join(choice((str.upper, str.lower))(c) for c in s)
+
+
+def log_in_user(client, user, password=TEST_USER_PASSWORD):
+    client.logout()
+    return client.post(reverse('user_management_limited_login'), {'username': user, 'password': password}, secure=True)
+
+
+def submit_form(client,
+                view_name,
+                data={},
+                success_url=None,
+                success_query=None,
+                form_keys=['form'],  # name of form objects in RequestContext returned with response
+                error_keys=[],  # keys that must appear in form error list
+                *args, **kwargs):
+    """
+        Post to a view.
+        success_url = url form should forward to after success
+        success_query = query that should return one object if form worked
+    """
```

Comment:
I'm glad you were able to port this utility! I was wondering if it was going to be a pain to redo. It's used extensively in other parts of the test suite, I think, so will come in handy!